packages/protocol/contracts/L1/hooks/AssignmentHook.sol

```diff
@@ -6,6 +6,7 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/utils/cryptography/SignatureChecker.sol";
 import "../../common/EssentialContract.sol";
 import "../../libs/LibAddress.sol";
+import "../libs/LibConstStrings.sol";
```

Comment:
Please also use this in DeployL1.s.sol.

Reply:
Ok, but in `DeployOnL1.s.sol` it can be used but only 2 times because the functions (`deployProxy`, `register` and `copy_register`) we are supplying the `“xxxx_yyy”` are `string` literals, not `bytes32` representation of them, so it would require change the input parameters , which not handy, as it is used for debugging (printing the string) purposes. But i changed that 2 params.